app/src/main/java/org/mozilla/fenix/library/history/HistoryFragmentStore.kt

```diff
@@ -144,7 +156,8 @@ data class HistoryFragmentState(
     val mode: Mode,
     val pendingDeletionItems: Set<PendingDeletionHistory>,
     val isEmpty: Boolean,
```

Comment:
added kdoc and wrote a comment for `HistoryFragmentStore`.

It might be part of the confusion comes from `HistoryFragmentStore` having `val items: List<History>`, which isn't used since our implementation is based on the pagination library. 
 
 I opened an [issue](https://github.com/mozilla-mobile/fenix/issues/26312) to clean up messy parts.